secrets/provider/kubernetes/config.go

```diff
@@ -86,6 +92,7 @@ var configSchema = environschema.Fields{
 var configDefaults = schema.Defaults{
 	usernameKey:               schema.Omit,
 	passwordKey:               schema.Omit,
+	serviceAccountKey:         "default",
```

Comment:
I prefer to make it required without a default value.
The refresh may fail if you forget to add the SA name and the default is the incorrect one.

Reply:
yeah ok